modules/dnsdist/README.md

```diff
@@ -0,0 +1,109 @@
+<!--
+title: "DNSdist monitoring with Netdata"
+custom_edit_url: https://github.com/netdata/go.d.plugin/edit/master/modules/dnsdist/README.md
+sidebar_label: "DNSdist"
+-->
+
+# DNSdist monitoring with Netdata
+
+[`DNS dist`](https://dnsdist.org/) is a highly DNS-, DoS- and abuse-aware loadbalancer. 
```

Comment:
```suggestion
[`DNSdist`](https://dnsdist.org/) is a highly DNS-, DoS- and abuse-aware loadbalancer. 
```

See https://www.open-xchange.com/products/dnsdist/

There is a link to that `open-xchange` site in the docs: https://dnsdist.org/#questions-requests-or-comments